connector/src/main/scala/com/microsoft/kusto/spark/utils/ExtendedKustoClient.scala

```diff
@@ -86,6 +94,10 @@ class ExtendedKustoClient(
         }
         tmpTableSchema = tableSchemaBuilder.toString
         executeEngine(database, generateTableCreateCommand(table, tmpTableSchema), crp)
+        if (writeOptions.writeMode == WriteMode.Stream) {
```

Comment:
That's a requirement question.
Data Connections can at times create tables, but I'd say user should have an Opt out or Opt in way to control this so it would not come as a surprise